README.md

```diff
@@ -139,37 +146,44 @@ Vanilla - /whitelist on
 ## Unlock books to re-edit
 
 ## Chairs
-CraftBook  
-BetterChairs
+[CraftBook](https://www.spigotmc.org/resources/craftbook.2083/)  
+[BetterChairs](https://www.spigotmc.org/resources/betterchairs-remastered.84809/)  
+[GSit](https://www.spigotmc.org/resources/gsit-modern-sit-seat-and-chair-lay-and-crawl-plugin-1-13-x-1-17-x.62325/)
 
 ## Backpacks
 
 ## Server MOTD
 You can simply use your server.properties file to achieve this.  
-ServerListPlus
+[ServerListPlus](https://www.spigotmc.org/resources/serverlistplus.241/)  
+[PistonMOTD](https://www.spigotmc.org/resources/pistonmotd-rgb-and-1-17-support.80567/)
 
 ## Elytra cost/control
+[Purpur](https://purpur.pl3x.net/)
```

Comment:
What option is this specifically? Thought it'd be a plugin lmao

Reply:
Lets you control how much damage elytra takes every second of flight and if it should take damage when using firework/trident [this option specifically](https://purpur.pl3x.net/docs/Configuration/#elytra)

Reply:
I see, considering it's likely something that doesn't work out of the box and Purpur is a whole new server type, it may be worth adding the config option on how to enable it.